src/test/resources/log4j2-test.yaml

```diff
@@ -0,0 +1,26 @@
+Configuration:
+  status: warn
+
+  Properties:
+    Property:
+      name: component_name
+      value: "audio-api"
+
+  Appenders:
+    Console:
+      name: Console
+      target: SYSTEM_OUT
+      PatternLayout:
+        Pattern: "[${component_name}] %d{yyyy-MM-dd HH:mm:ss.SSS} [%t] (%X{correlationID}) %-5level %C.%M#%L: %msg%n"
+
+  Loggers:
+    Logger:
+      name: "no.ndla"
+      level: error
+      additivity: false
+      AppenderRef:
+        - ref: Console
+    Root:
+      level: error
+      AppenderRef:
+        - ref: Console
```

Comment:
Trenger vi egentlig en så utbrodert log4j-konfigurasjon for testene våre?
Tror det vil være mer enn tilstrekkelig med følgende
```

Configuration:
  status: fatal

  Loggers:
    Root:
      level: fatal

```